### PR TITLE
Add net.sourceforge.liferea

### PR DESCRIPTION
**Pull request by mikelolasagasti**

### CODE REVIEW

**Comment by mikelolasagasti** [comment]
bot, build net.sourceforge.liferea

**Comment by mikelolasagasti** [comment]
bot, build net.sourceforge.liferea

**Comment by mikelolasagasti** [comment]
bot, build net.sourceforge.liferea

**Comment by mikelolasagasti** [comment]
bot, build net.sourceforge.liferea

File: net.sourceforge.liferea.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+    "app-id":"net.sourceforge.liferea",
+    "runtime":"org.gnome.Platform",
+    "runtime-version":"3.36",
```
Comment (LeandroStanger) [inline net.sourceforge.liferea.json:4]: Update runtime to 3.38
```suggestion
    "runtime-version":"3.38",
```

File: net.sourceforge.liferea.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            ]
+        },
+
+        {
+            "name":"liferea",
+            "sources":[
+                {
+                    "type":"git",
+                    "url":"https://github.com/mikelolasagasti/liferea.git",
+                    "branch":"flatpak"
```
Comment (LeandroStanger) [inline net.sourceforge.liferea.json:57]:
```suggestion
                    "commit":"2960fe4c77cca42a818cfc6b4282dfe0ef9fe503"
```

File: net.sourceforge.liferea.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
+{
+    "app-id":"net.sourceforge.liferea",
```
Comment (LeandroStanger) [inline net.sourceforge.liferea.json:2]:
```suggestion
    "app-id":"de.lzone.Liferea",
```

File: net.sourceforge.liferea.json [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "*.la",
+        "*.a"
+    ],
+    "finish-args":[
+        "--device=dri",
+        "--share=ipc",
+        "--socket=pulseaudio",
+        "--socket=x11",
```
Comment (barthalion) [inline net.sourceforge.liferea.json:19]:
```suggestion
        "--socket=fallback-x11",
```

File: net.sourceforge.liferea.json [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+                }
+            ]
+        },
+
+        {
+            "name":"liferea",
+            "sources":[
+                {
+                    "type":"git",
+                    "url":"https://github.com/mikelolasagasti/liferea.git",
```
Comment (barthalion) [inline net.sourceforge.liferea.json:56]: I understand this is your personal fork. Have you contacted upstream developers about submitting their app to Flathub?

Comment (mikelolasagasti) [reply net.sourceforge.liferea.json:56]: Correct. It is easier to test patches this way than manually adding patches as source.

And yes, I contacted Lars from Liferea about this request. My plan is to give him and whoever project requires access to resultant flathub repository.

File: net.sourceforge.liferea.json (hunk 3 above)
Comment (mikelolasagasti) [reply net.sourceforge.liferea.json:2]: app-id is net.sourceforge.liferea: https://github.com/lwindolf/liferea/blob/master/src/liferea_application.c#L345 or https://github.com/lwindolf/liferea/blob/master/liferea.appdata.xml.in#L3

**Comment by mikelolasagasti** [comment]
bot, build net.sourceforge.liferea

File: net.sourceforge.liferea.json [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+            ],
+            "sources":[
+                {
+                    "type":"archive",
+                    "url":"https://download.gnome.org/sources/libpeas/1.28/libpeas-1.28.0.tar.xz",
+                    "sha256":"42d91993b46ed50f16add6d9577ecc22beb8e2dffa7101e2232c2b63733b8b15"
+                }
+            ]
+        },
+        {
```
Comment (bilelmoussaoui) [inline net.sourceforge.liferea.json:40]: Use the one in shared-modules if possible.

**Comment by barthalion** [comment]
bot, build net.sourceforge.liferea

**Comment by barthalion** [comment]
bot, build net.sourceforge.liferea

File: net.sourceforge.liferea.json (hunk 5 above)
Comment (lwindolf) [reply net.sourceforge.liferea.json]: All fine from upstream :-)

**Pull request closed without merging**